src/main/resources/reference.conf

```diff
@@ -1263,23 +1263,29 @@ resourceTypes = {
       view_journal = {
         description = "View datasnapshot journal entries"
       }
+      read_auth_domain = {
+        description = "view the auth domain of the snapshot"
+      }
+      update_auth_domain = {
+        description = "update the groups in the auth domain of the snapshot"
```

Comment:
Is there potentially multiple auth domains for a snapshot or is there only one? If there's only one, I think this comment could be better described as "Update the auth domain of the snapshot" rather than mentioning groups. Just because auth domains are by definition defined by groups.

Reply:
Each resource only has one auth domain: https://sam.dsde-dev.broadinstitute.org/#/Resources/getAuthDomainV2